content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -113,9 +113,14 @@ If your project build directory is outside the project path, you must include th
         ```
 
         {{< important >}}
-If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor and other tools from the install directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
+If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the installed engine's build directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
         {{< /important >}}
 
 You can also run Project Manager (`o3de.exe`) from the same directory to edit your project's settings, add or remove Gems from the project, rebuild your project, and launch the Editor.
 
+
+{{< caution >}}
+When you launch the Editor, the **Asset Processor** from the same directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.
```

Comment:
Reword suggestion for clarity - When you launch the Editor, the **Asset Processor** that resides in that directory will be launched. Any existing **Asset Processor** tasks that are running must be closed before you attempt to launch the Editor from a different directory.

Reply:
I agree that this note could be improved, but this rewording still feels a bit awkward to me.

Reply:
Starting the second sentence with an infinitive phrase might help. For example, "To launch the Editor from a different directory, you must close any **Asset Processor** tasks that are running."

Reply:
> To launch the Editor from a different directory, you must close any **Asset Processor** tasks that are running.

I agree, this is a better rewording.  I'll make the change pending any other comments.

Reply:
Yeah that sounds good